packages/date-picker/index.ts

```diff
@@ -1 +1,3 @@
 export * from "./src/DatePicker"
+export * from "./src/DateRangePicker"
+export * from "./src/enums"
```

Comment:
```suggestion
export * from "./src/enums"
export * from "./src/types"
```

We should provide types too :)